sources/packages/forms/src/form-definitions/studentadditionaltransportationappeal.json

```diff
@@ -0,0 +1,1094 @@
+              "dataGridLabel": false,
+              "labelPosition": "top",
+              "description": "",
+              "errorLabel": "",
+              "tooltip": "",
+              "hideLabel": false,
+              "tabindex": "",
+              "disabled": false,
+              "autofocus": false,
+              "dbIndex": false,
+              "customDefaultValue": "",
+              "calculateValue": "",
+              "calculateServer": false,
+              "widget": null,
+              "attributes": {},
+              "validateOn": "change",
+              "validate": {
+                "required": true,
+                "custom": "",
+                "customPrivate": false,
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false,
+                "onlyAvailableItems": true
+              },
+              "overlay": {
+                "style": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+              },
+              "allowCalculateOverride": false,
+              "encrypted": false,
+              "showCharCount": false,
+              "showWordCount": false,
+              "properties": {},
+              "allowMultipleMasks": false,
+              "addons": [],
+              "inputType": "radio",
+              "fieldSet": false,
+              "id": "efqfe2q",
+              "tags": [],
+              "lockKey": true,
+              "isNew": false
+            },
+            {
+              "autofocus": false,
+              "input": true,
+              "tableView": false,
+              "inputType": "text",
+              "inputMask": "",
+              "label": "Please estimate your weekly transportation costs.",
+              "key": "additionalTransportCost",
+              "placeholder": "",
+              "prefix": "$",
+              "suffix": "",
+              "defaultValue": "",
+              "protected": false,
+              "persistent": true,
+              "hidden": false,
+              "clearOnHide": true,
+              "delimiter": true,
+              "decimalLimit": 2,
+              "requireDecimal": true,
+              "validate": {
+                "required": true,
+                "multiple": "",
+                "custom": "valid = (input >= 0 && input <= 9999) ? true : 'The number you have entered is outside the expected range.';"
+              },
+              "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+              },
+              "type": "currency",
+              "labelPosition": "top",
+              "tags": [],
+              "properties": {},
+              "lockKey": true,
+              "customConditional": "show = (data.transportationCostSituation === 'noLimit' || data.transportationCostSituation === 'special' || data.transportationCostSituation === 'educationPlacement');"
```

Comment:
This conditional seems to be present in the Student Application in only two places but in this new form it is present six times. Is it expected?
It seems, for instance, that the uploaded and its container both have the same show `logic`. Can you please check?

Reply:
I added a container and moved the condition to the container only.

Reply:
The container component is not only for organization and it changes the components hierarchy. Please check if the submitted data has not changed. When there is a need for a container for the only sake of applying a conditional and not affecting the component we can use a regular panel with a `custom CSS class` defined as `formio-panel-unset`.

Reply:
Fixed.